src/main/webapp/feedback.jsp

```diff
@@ -0,0 +1,76 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%
+  Boolean canOpen = (Boolean) request.getAttribute("feedbackOpen");
+  String role = (String) request.getAttribute("role");
+  pageContext.setAttribute("feedbackOpen", canOpen);
+  pageContext.setAttribute("role", role); 
+%>
+
+
+<c:choose>
+  <c:when test= "${canOpen == false}">
```

Comment:
Nit: change to ! canOpen

Reply:
When changed to !canOpen instead of canOpen == false, the conditional acts as if the value is true and displays the message when it should display the form